src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -50,21 +55,26 @@ struct NormOneBridge{T} <: AbstractBridge
     eq_index::CI{MOI.ScalarAffineFunction{T}, MOI.EqualTo{T}}
     ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
 end
-function bridge_constraint(::Type{NormOneBridge{T}}, model, f::MOI.VectorOfVariables, s::MOI.NormOneCone) where T
-    return bridge_constraint(NormOneBridge{T}, model, MOI.VectorAffineFunction{T}(f), s)
-end
-function bridge_constraint(::Type{NormOneBridge{T}}, model, f::MOI.VectorAffineFunction{T}, s::MOI.NormOneCone) where T
+function bridge_constraint(::Type{NormOneBridge{T}}, model, f::MOI.AbstractVectorFunction, s::MOI.NormOneCone) where T
+    if f isa MOI.VectorOfVariables # TODO currently needed for test to pass
```

Comment:
@blegat this may have exposed a bug? without these 3 lines, normone1v test fails (though normone1f test passes).

Reply:
Indeed, MWE welcome :)